package.json

```diff
@@ -6,21 +6,39 @@
     },
     "scripts": {
         "build": "webpack --config webpack.config.js --mode development",
-        "test": "echo \"Error: no test specified\" && exit 1"
+        "test": "jest"
     },
     "repository": {
         "type": "git",
         "url": "git+https://github.com/scpwiki/wikijump.git"
     },
     "browserslist": "cover 99.5%",
+    "babel": {
+        "presets": [
+            [ "@babel/preset-env", { "targets": {"node": "current"} } ],
+            "@babel/preset-typescript"
+        ]
+    },
+    "jest": {
+        "moduleFileExtensions": ["js", "ts"],
+        "moduleNameMapper": {
+            "^@/(.*)$": "<rootDir>/web/files--common/$1"
```

Comment:
Curious, what does this do?

Reply:
Throughout the typescript I'm using `@` as an alias for `./web/files--common`, so that I can do `import whatever from "@/javascript/thing"` (and eventually `"@/modules/thing"`)

https://github.com/scpwiki/wikijump/blob/197ce822449b94b0428fb428615ceda42d045953/webpack.config.js#L23-L24

https://github.com/scpwiki/wikijump/blob/197ce822449b94b0428fb428615ceda42d045953/tsconfig.json#L19-L20

That just sets it up so that Jest understands that alias, too, but for Jest it's a bit lower-level and needs a regex replacement.

Reply:
I would consider changing it to `@web` in case we need to add other special paths in the future. But this is fine.